server/utils/token.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line func-names
+const generateToken = function (i) {
+  let rnd = '';
```

Comment:
Имя переменной rnd не говорит ничего о назначении данной переменой. Лучше не использовать сокращения

Reply:
```suggestion
  cont token = generateRandomString(size);
```
Лучше вместо цикла сделать рекурсию и вынести в отдельную функцию.
Переменную тоже лучше не называть `i`, складывается ощущение, что тут где-то счетчик цикла. А на самом деле это длина токена